test/contracts/application/versioning/FreezableImplementationDirectory.test.js

```diff
@@ -70,4 +70,20 @@ contract('FreezableImplementationDirectory', ([_, owner, anotherAddress]) => {
       })
     })
   })
+
+  describe('unsetImplementation', function () {
+    describe('when it is not frozen', function () {
```

Comment:
This `describe` block doesn't seem to be testing anything different to the one within the `setImplementation` describe block. Remove it if so.